pkg/logs/config/integration_config.go

```diff
@@ -254,3 +293,17 @@ func BuildTagsPayload(configTags, source, sourceCategory string) []byte {
 
 	return tagsPayload
 }
+
+// buildSourceStatus returns the status of source
+func buildSourceStatus(source IntegrationConfigLogSource) status.Source {
+	var info string
+	switch source.Type {
+	case FileType:
+		info = fmt.Sprintf("tailing file(s) matching %s", source.Path)
+	case DockerType:
+		info = fmt.Sprintf("tailing docker image %s", source.Image)
+	case TCPType, UDPType:
+		info = fmt.Sprintf("listenning on port %d", source.Port)
```

Comment:
typo: `listenning` -> `listening`

Reply:
You're right, I've just fixed that thanks.